packages/js/examples/vanilla-calling/index.html

```diff
@@ -134,7 +134,7 @@ <h4>Troubleshooting</h4>
       // client.autoRecoverCalls = false;
       client.remoteElement = 'remoteVideo';
       client.localElement = 'localVideo';
-
+      client.iceServers = [{ urls: ['stun:stun.l.google.com:19302'] }];
```

Comment:
should this be in an environment variable?

Reply:
This is an example file for users to test the SDK